model/deployment.go

```diff
@@ -16,12 +16,13 @@ package model
 
 import (
 	"encoding/json"
-	"github.com/pkg/errors"
 	"time"
 
-	"github.com/go-ozzo/ozzo-validation/v4"
+	"github.com/pkg/errors"
+
+	validation "github.com/go-ozzo/ozzo-validation/v4"
 	"github.com/go-ozzo/ozzo-validation/v4/is"
-	"github.com/satori/go.uuid"
+	uuid "github.com/satori/go.uuid"
```

Comment:
Not really related to the pr, but can we change this one to github.com/google/uuid ? (ref. satori/go.uuid#115)

Reply:
Yes we can, do you want me to do it now?

Reply:
If you have time, otherwise I can do it later.

Reply:
I will do it as a follow-up PR, then.